draft-ietf-protected-audience-key-value-services.md

```diff
@@ -579,9 +624,52 @@ The output is a `response` to be sent to a Client.
    `rctxt`, as described in {{response-encryption}}. Return an empty `response` on failure.
 9.  Return `response`.
 
-### Parsing a Response
-
-TODO
+### Parsing a Response (#response-parsing)
+
+This section describes how a conforming Client MUST parse and validate
+a response from a Trusted Key Value service.
+
+It takes as input the `request context` returned from
+{{request-generate}} in addition to the `encrypted response`.
```

Comment:
I wonder if you also want to indicate here that this is an example that only is looking to accept no compression or gzip (0 or 2), which is why it checks for 0 or 2. 

Or you may want to change the language from "Client MUST" to "Client MAY". Otherwise it makes it seem like all clients MUST rejetc compression bit 1.

Reply:
But don't we want to reject the the responses with compression bit as 1 for now?

Reply:
Oh, I wasn't aware.